tools/cabana/chartswidget.cc

```diff
@@ -166,8 +165,11 @@ void ChartsWidget::setMaxChartRange(int value) {
 }
 
 void ChartsWidget::updateToolBar() {
+  if (docking) setColumnCount(0);
```

Comment:
I think it would be better not to set it to 0 and allow multiple columns when docking,  let the charts automatically adjust according to the settings,it's a good feature for wide screen:

[Kazam_screencast_00068.webm](https://user-images.githubusercontent.com/27770/215329793-8f79d6e8-2476-4112-9aff-d615437b8624.webm)